server/server.js

```diff
@@ -308,6 +308,37 @@ app.post(`${apiBaseUrl}/participants`,
     }
   }));
 
+app.post(`${apiBaseUrl}/participants`,
+  asyncMiddleware(async (req, res) => {
+    await validate(ParticipantSchema, req.body);
+
+    const participant = {
+      ...req.body,
+      maximusId: null,
+      postalCodeFsa: req.body.postalCode.substr(0, 3),
+      preferredLocation: req.body.preferredLocation.join(';'),
+      interested: true,
+      nonHCAP: null,
+      crcClear: null,
+      callbackStatus: false,
+      userUpdatedAt: new Date().toJSON(),
+    };
+    delete participant.consent; // Not persisted to DB
```

Comment:
I think we actually have an obligation to save this - not losing any user-submitted data is important for FOI requests, even if the user by definition can't submit without this being true